.github/workflows/publish.yml

```diff
@@ -38,4 +38,10 @@ jobs:
               run: npm run ci:build
 
             - name: Commit version
-              run: git add dist -f && git commit --amend --no-edit dist && git push --follow-tags
+              run: |
+                git add dist -f && \
+                git commit --amend --no-edit dist && \
+                TAG=`git describe --tags $(git rev-list --tags --max-count=1)` && \
```

Comment:
`git commit --amend` changes commit hash, so release is pointing to commit, which hash is not in the tree
`git describe --tags $(git rev-list --tags --max-count=1)` returns last tag and saves it into `TAG`
`git tag -d $TAG` && `git tag $TAG` moves this release tag to the last commit (with release)